### ISSUE TITLE
SyntaxError: Failed to execute 'querySelector' on 'Element

### ISSUE THREAD

**Issue by peterhaaf** [jdbranham/grafana-diagram#175]
Hi,

i am trying to use Version 1.7.1 of grafana-diagram on grafana Version: 7.2.1

I have a prometheus query: probe_duration_seconds{instance="https://www.bing.com/"}. Results are displayed as:
probe_duration_seconds_instance__https_//www.bing.com/__ job__blackbox-web__ in the legend area, but the diagram reports an error: 
Error rendering diagram. Check the diagram definition

SyntaxError: Failed to execute 'querySelector' on 'Element': '#probe_duration_seconds_instance__https_//www.bing.com/__ job__blackbox-web__' is not a valid selector.

The diagram definition makes no use of this metric ( test stage ), it is very simple: 
graph LR 
   A --> B
So i assume, this error has nothing to do with my diagram definition., but i have no idea, what goes wrong, please help

Thank you very much, best regards
Peter

**Comment by jdbranham** [comment]
Looks like the generated element ID is invalid because of the forward slashes.  
Can you modify the alias for your query to eliminate the `/` ?

**Comment by peterhaaf** [comment]
Hi 

as a workaround, indeed, i aliased the query without the URL, that came out of the query and the grafana-diagram worked. 
But this is not very nice, as the URL is part of the query result. ( see Blackbox-Exporter -> Prometheus ), so need to define a query for each URL i want to display, instead of one query and have let diagram plugin taking care 
Note: This used to work, in older Versions of the diagram plugin, that`s how i identified this issue 

Peter

**Comment by jdbranham** [comment]
Thanks Peter - I'll fix this.

**Comment by xiaopeng163** [comment]
same issue found

```
Panel Title
Error rendering diagram. Check the diagram definition

SyntaxError: Failed to execute 'querySelector' on 'Element': '#Port-channel10.100' is not a valid selector.
```


```
graph LR
      A -->|Port-channel10.100| B
```

**Comment by jdbranham** [comment]
Note -  
Grafana also has a regex rename for metrics as part of the 'Transformation' options now.   
In this case -  
`probe_duration_seconds_instance__https_//www.bing.com/__ job__blackbox-web__` 
becomes -  
`_www.bing.com___ job__blackbox-web__ ` 


![image](https://user-images.githubusercontent.com/2660839/120828059-b408cb80-c521-11eb-8037-910882e5e02f.png)

**Comment by jdbranham** [comment]
Fixed in 1b24a373da95dc38f6b1322d2e287cf9efea73c5

v1.7.3
